hardhat.config.ts

```diff
@@ -132,7 +136,7 @@ const config: HardhatUserConfig = {
     },
     arbitrum: {
       chainId: CHAIN_IDs.ARBITRUM,
-      url: getNodeUrl("arbitrum", true, CHAIN_IDs.ARBITRUM),
+      url: getNodeUrl("arbitrum-mainnet", true, CHAIN_IDs.ARBITRUM),
```

Comment:
these were renamed by infura?

Reply:
Yeah they must have been. They now follow a naming convention given by `[network_name]-[mainnet or sepolia].infura.io.`

Reply:
huh ok that makes sense, does this just not work currently with the renamed network names?